contracts/JsmnSolLib.sol

```diff
@@ -127,9 +132,9 @@ library JsmnSolLib {
             parser.pos = start;
             return RETURN_ERROR_PART;
         }
-
+        
         // found the end
-        var (success, token) = allocateToken(parser, tokens);
+        (success, token) = allocateToken(parser, tokens);
```

Comment:
I've read up on 0.4.24, which allows for multi variable declarations. Thus, I would prefer `(bool success, Token token)`so that the vars are clearly defined. What do you think?

Reply:
yes I actually rectified it but forgot to commit it I guess. anyway committing it with the text case fix as well